test/lint/commit-script-check.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env sh
```

Comment:
Missing expected shebang "#!/usr/bin/env bash" or "#!/bin/sh" in test/lint/commit-script-check.sh